ServerSide/yoty/Core/Managers/Bids/IBidsManager.cs

```diff
@@ -1,20 +1,20 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using YOTY.Service.WebApi.PublicDataSchemas;
+﻿// Copyright (c) YOTY Corporation and contributors. All rights reserved.
 
-namespace YOTY.Service.Managers.Bids
+namespace YOTY.Service.Core.Managers.Bids
 {
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using YOTY.Service.WebApi.PublicDataSchemas;
+
     public interface IBidsManager
     {
         // create bid
         Task<Response<BidDTO>> CreateNewBid(NewBidRequst productBid);
 
         // get bid details
         Task<Response<BidDTO>> GetBid(string bidId);
-        Task<IList<BuyerDTO>> GetBidBuyers(string bidId);
-        Task<IList<BuyerDTO>> GetBidSuplliers(string bidId);
+        Task<Response<List<BuyerDTO>>> GetBidBuyers(string bidId);
+        Task<Response<List<SupplierProposalDTO>>> GetBidSuplliersProposals(string bidId);
```

Comment:
IList

Reply:
@yotamc4  i  remember iv'e change it in purpose but I don't remember why ... changing it back to IList even though I think it can make some troubles